agent/taskresource/volume/dockervolume.go

```diff
@@ -332,11 +345,11 @@ func (vol *VolumeResource) Create() error {
 func (vol *VolumeResource) Cleanup() error {
 	// Enable volume clean up if it's task scoped
 	if vol.VolumeConfig.Scope != TaskScope {
-		seelog.Debugf("Volume [%s] is shared, not removing", vol.Name)
+		vol.log.Infof("Volume is shared, not removing")
```

Comment:
we should keep the volume identifier here to reduce scrolling up and down the logs.

Reply:
got updated on the design, dismiss this comment please.